### PR TITLE
add changes tru PR

### PR DESCRIPTION
**Pull request by delima-justine**
- Add line of text in the file

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Git Branch Demo
 main branch change
-feature-1 change
\ No newline at end of file
+feature-1 change
+add change tru PR
```
Comment (justine-skie) [inline file.txt]: Change tru to through

**Review by justine-skie — CHANGES_REQUESTED** [review]
Please refer to the coding [guidelines](https://docs.google.com/presentation/d/1INQFb9d0Ajyq-FGkKR2uY27VT6iscjGoj0KgjXOVlCo/edit#slide=id.g30ae8cef28b_0_125). I will approve the pull request once changed.

File: file.txt (hunk 1 above)
Comment (justine-d5) [reply file.txt]: Fixed in commit [03b388baa0b3350eadc95855cdbfdf43b7dd9b04]

Comment (delima-justine) [reply file.txt]: Fixed in commit 03b388baa0b3350eadc95855cdbfdf43b7dd9b04

**Pull request merged**